aqua/MenuMeters/Portfile

```diff
@@ -17,18 +18,13 @@ long_description    The MenuMeters monitors are true SystemUIServer plugins
                     (also known as Menu Extras). This means they can be reordered   \
                     using command-drag and remember their positions in the menubar  \
                     across logins and restarts.
+
 homepage            http://www.ragingmenace.com/software/menumeters
```

Comment:
I was just considering that also. I think it makes sense to point to the new one given that the old page hasn't been touched since 2015. And the new page has an issue tracker many people have been using.